lib/sshnp.dart

```diff
@@ -545,6 +553,7 @@ class SSHNP {
         localSshOptions: results['local-ssh-options'] ?? [],
         rsa: results['rsa'],
         sendSshPublicKey: sendSshPublicKey,
+        remoteUsername: remoteUsername,
```

Comment:
change to `remoteUsername: results['remote-user-name']`,

Reply:
Updated